### PR TITLE
Cleanup upgrade tags that have optimized

### PR DESCRIPTION
**Pull request by liangxia**
Follow up of https://github.com/openshift/release/pull/49306

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml [hunk 1]
```diff
@@ -340,38 +294,27 @@ tests:
...
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.13 and @azure-upi and not @fips'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.13 and @azure-upi and not @fips'
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
 - as: gcp-ipi-disc-priv-oidc-f28
   cron: 57 21 23 * *
   steps:
     cluster_profile: gcp-qe
-    env:
```
Comment (jianlinliu) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:352]: Shall we add `@disconnected` tag here?

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:352]: Yes, we should. 
This PR is cleanup tags that have been optimized and it's already large enough. I think add missed `@disconnected` tag is another task that might be better in a follow up PR.

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:352]: Address the issue in https://github.com/openshift/release/pull/49468

**Pull request merged**